includes/Admin/Editor.php

```diff
@@ -387,6 +387,20 @@ public function get_editor_settings(): array {
 				}
 			}
 		}
+		/**
+		 * Revision.
+		 *
+		 * @var int $revision
+		 */
+		$revision = isset( $_GET['revision'] ) ? absint( $_GET['revision'] ) : 0; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+
+		$revision_message = ! empty( $revision ) ?
+			sprintf(
+				/* translators: %s: Number of stories. */
+				__( 'Post restored to revision from %s.', 'web-stories' ),
+				wp_post_revision_title( $revision, false )
```

Comment:
It is a little hard to pass this in the rest api. So cleaner to just pass the whole message down.